### ISSUE TITLE
Would you be open to changing the name?

### ISSUE THREAD

**Issue by claym** [jswanner/DontFuckWithPaste#15]
My employer is implementing a white list for Chrome extensions, and I would love to submit yours - I use it and love it.

However, the name doesn't really fit in a Fortune 50 company :D 

I know it's stupid, but was just hoping it would be something you'd consider, or maybe stick up a duplicate with a more friendly name.

Regardless, thanks!

**Comment by jswanner** [comment]
Sorry for not replying to this issue earlier, I wasn't just ignoring your request, but was really spending a lot more time thinking about it than you'd probably believe.

**Short Answer**
I'm sorry, I'm not willing to change the name, nor release it under another name.

**Longer Answer**
I wrote this extension as my own personal protest against my fellow web developers, who for whatever reason, have decided to try to take more control over their users' experience while using their site. I don't agree with this heavy-handed approach over a person's browsing experience, and so created this extension in protest to that. To me, the name is part of that protest, and changing it would represent a shift of purpose for this extension: from ideological to practical.

I understand many are using this extension for more practical purposes, with little to no concern of the ideological. And, I also understand that when I choose not to accept a change for its practical benefits, because it does not reflect my ideology, that I'm being selfish. But, that's why I released this code under an extremely permissive license, so that anyone else can take it all and do whatever they would like with it, like releasing it under a different name.
